discord_bots/cogs/admin.py

```diff
@@ -0,0 +1,692 @@
+                ).delete()
+                # TODO: Test this part
+                queue_waitlist: QueueWaitlist | None = (
+                    session.query(QueueWaitlist)
+                    .filter(
+                        QueueWaitlist.queue_id == queue.id,
+                    )
+                    .first()
+                )
+                if queue_waitlist:
+                    session.query(QueueWaitlistPlayer).filter(
+                        QueueWaitlistPlayer.player_id == member.id,
+                        QueueWaitlistPlayer.queue_waitlist_id == queue_waitlist.id,
+                    ).delete()
+
+            queue_statuses = []
+            queue: Queue
+            for queue in session.query(Queue).order_by(Queue.created_at.asc()).all():  # type: ignore
+                queue_players = (
+                    session.query(QueuePlayer)
+                    .filter(QueuePlayer.queue_id == queue.id)
+                    .all()
+                )
+                queue_statuses.append(
+                    f"{queue.name} [{len(queue_players)}/{queue.size}]"
+                )
+            session.commit()
+
+        await interaction.response.send_message(
+            embed=Embed(
+                title=f"{escape_markdown(member.name)} removed from: {', '.join([queue.name for queue in queues])}",
+                description=" ".join(queue_statuses),
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(name="editcommand", description="Edit a custom command")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(
+        name="Name of existing custom command", output="New command output"
+    )
+    async def editcommand(self, interaction: Interaction, name: str, *, output: str):
+        session: SQLAlchemySession
+        with Session() as session:
+            exists = (
+                session.query(CustomCommand).filter(CustomCommand.name == name).first()
+            )
+            if exists is None:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description="Could not find a command with that name",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+
+            exists.output = output
+            session.commit()
+
+        await interaction.response.send_message(
+            embed=Embed(
+                description=f"Command `{name}` updated",
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(
+        name="editgamewinner", description="Edit the winner of a finished game"
+    )
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(game_id="Finished game id", outcome="<tie|be|ds>")
```

Comment:
does `outcome` need to be a string too or can it also be a list of `Literals`?

Reply:
Not sure if I'm misunderstanding, but `outcome` already is a Literal list of of strings.
`outcome: Literal["Tie", "BE", "DS"]`

Reply:
I think I misunderstood myself, since this is just for the description. I don't think outcome even needs a description, since it will be autofilled to 3 valid options: "Tie", "BE", or "DS" -- my point was that this doesn't match precisely

Reply:
Understood.
I do prefer to have the `.describe()` there for it anyway, just for completeness sake. I have updated it thought to match the `Literal` (`"<Tie|BE|DS>"`)

Reply:
Updated description to `"Tie, BE, DS"` as discussed in Discord